node/config.go

```diff
@@ -190,6 +190,13 @@ type Config struct {
 
 	// AllowUnprotectedTxs allows non EIP-155 protected transactions to be send over RPC.
 	AllowUnprotectedTxs bool `toml:",omitempty"`
+
+	// Endpoint of L1 HTTP-RPC server
+	L1Endpoint string `toml:",omitempty"`
+	// Number of confirmations on L1 needed for finalization
+	L1Confirmations int `toml:",omitempty"`
```

Comment:
Can we also support `safe` and `finalized` as we do in scroll? There we use `rpc.BlockNumber` for this (but I think that only accepts hex for numbers, might be inconvenient to use)